### PR TITLE
Detective now spawns with a police baton and Sec Wardrobe gets black gloves

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request

Detective now gets a police baton! ( /tg/ has done this also)

Security Wardrobes now start with 3 pairs of black gloves. So deputies / new recruits don't leave their fingerprints at crime scenes.

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
does it remove their telebaton or something

**Comment by ParagonPrime** [comment]
Hmm, would still be better to give him a secHUD, but alright.

**Comment by Super3222** [comment]
He's not an officer and shouldn't be doing any arresting. He's an investigator.

**Comment by ParagonPrime** [comment]
yes, but it can still be useful and i don't particularly see how it encourages det-officering

**Comment by FluffySurvivor** [comment]
that means you have to add something unique and powerful to the derelict to compensate
otherwise no.

**Comment by ShadowDeath6** [comment]
The police baton on the derelict is already stupidly powerful

**Comment by Super3222** [comment]
I could argue that the telebaton is too OP for every head to have, since it takes 10 hits to kill them.

Telebatons, Toolboxes, etc do 10 brute damage therefore needing 10 hits.

Hatchets, big crowbar, and police batons do 12 brute damage and take 9 hits. WOW! HOW OVERPOWERED!!

These aren't doing 30 brute damage like an energy sword. Get over it.

And you should notice that the Detective is a resemblance to the old 1990's detectives, minus the forensic scanner. This isn't that big of a change.

File: html/changelogs/Super3222-SecUpdates.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Nanotrasen now admissions a police baton to their Detectives for clubbing criminals to the curb."
+  - rscdel: "Security Wardrobes now have black gloves in them."
```
Comment (TehFlaminTaco) [inline html/changelogs/Super3222-SecUpdates.yml]: rscdel?

Do you mean rscadd?

Comment (Super3222) [inline html/changelogs/Super3222-SecUpdates.yml]: Yes.

**Pull request merged**